server/package.json

```
@@ -21,6 +21,7 @@
     "test:integration": "node --max-old-space-size=4096 --experimental-vm-modules node_modules/jest/bin/jest.js --forceExit --ci tests/integration.test.js",
     "test:related:jest": "node --max-old-space-size=4096 --experimental-vm-modules node_modules/jest/bin/jest.js --forceExit --runInBand --passWithNoTests --findRelatedTests",
     "test:related:vitest": "vitest related",
+    "test:unit": "vitest run tests/OrganizationService.test.js tests/InvitationService.test.js tests/organizationController.test.js tests/knowledgeController.test.js tests/transcriptController.test.js tests/meetingDigestService.test.js tests/MeetingService.test.js tests/sharedLinkAnalytics.test.js tests/clerkRbacIntegration.test.js tests/decisionGraphOptimized.test.js tests/sessionAiPermission.test.js tests/routeRegistration.test.js tests/meetingPresenceSpoofing.test.js tests/pollExpirationJob.test.js tests/vectorSearchOrgFiltering.test.js tests/vectorMetadataMinimization.test.js tests/bookmarkOrphanedSafety.test.js tests/liveMeetingInvitationValidation.test.js tests/calendarConnectionSync.test.js tests/translationSocketAuth.test.js tests/knowledgeSearchRegexSafety.test.js tests/periodBucket.test.js tests/rateLimitStore.test.js tests/attachmentDownloadHeaders.test.js tests/translationAuthorization.test.js tests/analyticsRouteWiring.test.js tests/icebreaker.test.js",
```

Biome (2.5.7)

[error] The key test:unit was already declared.

(lint/suspicious/noDuplicateObjectKeys)